python/tach/cli.py

```diff
@@ -532,7 +554,7 @@ def tach_sync(
             exclude_paths=exclude_paths,
         )
     except Exception as e:
-        print(str(e))
+        raise e
         sys.exit(1)
```

Comment:
I think if we intend to simply re-raise `e` we should just eliminate the try/except entirely, and in my mental model the `sys.exit` won't ever be executed due to the re-raise anyway. But is there another motivation for this change?